### DISCUSSION TITLE
Tracking request?

### DISCUSSION THREAD

**Discussion by GogetaX** [Poing-Studios/godot-admob-editor#25] [Q&A] [answered]
Hello, 
I have published same game on Android and also on iOS with admob.
the Android version uploaded fine, no problems,
but on iOS: since they test each and every app i put on the App market, i got an email from them
the email goes something like:
"Your app uses the AppTrackingTransparency framework, but we are unable to locate the App Tracking Transparency permission request when reviewed on iOS 15.4."

what exactly does that mean?
as far as i understand, i have to put some kind of tracking request on my app as soon as it starts, and if tracking is granted, i use Personalized ads, if no, i use random ads (witch is handled by google, correct?)

so i did little bit of research, on admob website, there is GDPR button, i had to enable and select both of my ios and android platforms with my app.

it looks by default like this:
![image](https://user-images.githubusercontent.com/51264046/161059366-ed34e2cc-ace7-4184-b5b8-e4c574c5b50d.png)

so the main question is now:
how do i bring this popup at game start, is it something Admob plugin can handle or I have to call different library?
or im doing everything wrong? lol

**Comment by gumaciel** [comment, marked as answer]
Hello, we are not using `AppTrackingTransparency framework`, we are using `UserMessagingPlatform` to request ATT: 

https://developers.google.com/admob/ios/ios14#request

Read more here: https://support.google.com/admob/answer/10115027?hl=en

**Comment by GogetaX** [comment]
thank you very much.
i will see what i can do with this :)

i think i figured it out already..

**Reply by fuzuli-gulmammadli** [reply]
Hi I have the same issue. Can you please tell me how did you solve this issue?
